README.md

```diff
@@ -0,0 +1,56 @@
+# UEFI provisioning with Libvirt
+
+(Tested on Fedora 39)
+
+## Preconditions
+
+Required packages
+
+```console
+dnf install @virtualization dnf-plugins-core
+dnf config-manager --add-repo http://www.kraxel.org/repos/firmware.repo
+dnf install edk2.git-ovmf-x64
+```
+
+Enable Libvirt
+
+```console
+systemctl start libvirtd
+systemctl enable libvirtd
```

Comment:
You can do both in a single command. And you indicate it's console, so you can use `#` (root) or `$` (non-root) to indicate it's an actual command.
```suggestion
# systemctl enable --now libvirtd
```